packages/components/src/components/domainproperties/issues/IssuesListDefPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,301 @@
+        });
+
+        getCoreUsersInGroups().then((coreUsersData: List<UserGroup>) => {
+            this.setState(() => ({
+                coreUsers: coreUsersData,
+            }));
+        });
+    }
+
+    render() {
+        const { model, onSelect } = this.props;
+        const { coreUsers, coreGroups } = this.state;
+        return (
+            <Col xs={12} md={6}>
+                <SectionHeading title="Assignment Options" />
+                <AssignedToGroupInput
+                    model={model}
+                    coreGroups={coreGroups}
+                    onSelect={onSelect}
+                />
+                <DefaultUserAssignmentInput model={model} coreUsers={coreUsers} onSelect={onSelect} />
+            </Col>
+        );
+    }
+}
+
+export class SingularItemNameInput extends React.PureComponent<IssuesListDefBasicPropertiesInputsProps> {
+    getIssueNameHelpTip() {
+        return ISSUES_LIST_DEF_SINGULAR_PLURAL_TIP;
+    }
+    render() {
+        const { model, onInputChange } = this.props;
+        const value = model.singularItemName === null ? '' : model.singularItemName;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Singular item name" helpTipBody={this.getIssueNameHelpTip} required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        id="singularItemName"
+                        type="text"
+                        value={value}
+                        onChange={onInputChange}
+                    />
+                </Col>
+
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class PluralItemNameInput extends React.PureComponent<IssuesListDefBasicPropertiesInputsProps> {
+
+    getIssueNameHelpTip() {
+        return ISSUES_LIST_DEF_SINGULAR_PLURAL_TIP;
+    }
+
+    render() {
+        const { model, onInputChange } = this.props;
+        const value = model.pluralItemName === null ? '' : model.pluralItemName;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Plural items name" helpTipBody={this.getIssueNameHelpTip} required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        id="pluralItemName"
+                        type="text"
+                        value={value}
+                        onChange={onInputChange}
+                    />
+                </Col>
+
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class CommentSortDirectionDropDown extends React.PureComponent<IssuesListDefBasicPropertiesInputsProps> {
+
+    getHelpTip() {
+        return ISSUES_LIST_DEF_SORT_DIRECTION_TIP;
+    }
+
+    onChange = (name: string, formValue: any, selected: String, ref: any): any => {
```

Comment:
type change from String -> string here too:
```suggestion
    onChange = (name: string, formValue: any, selected: string, ref: any): any => {
```

Reply:
actually for this one, looks like `selected` isn't a string but an object so maybe just use `any` here?

Reply:
using 'any' for selected.